js/listdates.js

```diff
@@ -1,87 +1,58 @@
 /* global moment, humanizeDuration */
 var content = "";
```

Comment:
Makes sense — this script _is_ loaded at the bottom, but perhaps a domready would be a good sanity check for the fictional scenario where I open-source this as a JS library.